client/client.go

```diff
@@ -101,6 +101,19 @@ func (c *CSAPI) DownloadContent(t ct.TestLike, mxcUri string) ([]byte, string) {
 	return b, contentType
 }
 
+// DownloadContentV2 downloads media from _matrix/client/v1/media resource, returning the raw bytes and the Content-Type. Fails the test on error.
+func (c *CSAPI) DownloadContentV2(t ct.TestLike, mxcUri string) ([]byte, string) {
```

Comment:
Wondering if this should be something like `DownloadContentAuthenticated` or similar?
Having `V2` and then calling a `v1` endpoint may be a bit confusing.